pkg/utils/clusterinfo/fetcher.go

```diff
@@ -351,6 +351,10 @@ func storeStateToStatus(state string) ComponentStatus {
 		return ComponentStatusTombstone
 	case "offline":
 		return ComponentStatusOffline
+	case "down":
+		return ComponentStatusDown
+	case "disconnected":
+		return ComponentStatusDisconnected
```

Comment:
disconnected should be mapped to unreachable. For default maybe we should add Unknown status.